README.md

```diff
@@ -38,84 +41,44 @@ const userInformation = {
 
-So when our server reads a cookie from an incoming request, how can we be sure that the cookie has not been edited?
+So when our server reads a cookie from an incoming request, **how can we be sure that the cookie has not been edited?**
 
-### What is signing?
+## Signing
 
-It is possible to use hashing (the same thing we used to protect our passwords in [Workshop 1](https://github.com/foundersandcoders/ws-password-management)), in order to protect our cookie from being altered.
+We can use hashing (the same thing we used to protect our passwords in the password management [workshop](https://github.com/foundersandcoders/ws-password-management)), in order to know if our cookie has been altered.
 
-In previous hashes we just had to compare the results, password to password. This is known as 'integrity' (ie, is the data the same). But for our cookie this isn't enough, we also need to verify the 'authentication' (ie, did we create the hash).
+However, we need to be sure only we can produce the correct hash. So if an attacker changes the cookie, they cannot produce the correct hash, and we will know the data has changed.
 
-A [HMAC](https://en.wikipedia.org/wiki/Hash-based_message_authentication_code) (Hash-based message authentication code) is a way to hash a message in order to verify its integrity and authentication. This is important as we need to be sure that:
-1. We created the hash.
-2. The message has not changed.
+A [HMAC](https://en.wikipedia.org/wiki/Hash-based_message_authentication_code) (Hash-based message authentication code) is a code which authenticates a message using a hash. In other words, it is exactly what we need!
 
-A HMAC requires a `secret` (random string), a `value` (the string you want to protect) and `mathematical algorithm` to apply to them.
+A HMAC requires:
+- a `secret`: a long random string which is private,
+- a `value`: the message you want to sign, and
+- a `mathematical algorithm` to create the hash.
 
-These same three inputs will always produce the _same result_. So you can store the HMAC alongside the original message to verify in future, that the message/cookie/whatever has not been tampered with. This is known as 'signing'.
+You can store the HMAC alongside the original message to verify that the message/cookie/whatever has not been tampered with. This is known as **signing**.
 
-Luckily Node.js has a built-in [HMAC function](https://nodejs.org/dist/latest-v8.x/docs/api/crypto.html#crypto_class_hmac).
+**Metaphor Alert:** Think of the signature like a personal wax seal on a letter- the contents of the letter cannot be changed without breaking the seal, and the correct seal cannot be reproduced without the stamp!
 
-Note: When protecting a cookie, defence against brute force attacks (such as `bcrypt`) is not necessary, for two reasons:
+**Note:** When protecting a cookie, defence against brute force attacks (e.g. hashing with `bcrypt`) is not necessary, for two reasons:
```

Comment:
I have changed this note slightly to remove the confusing bit. I think there is potentially room for a faq section in this workshop or in the auth wekk in general, answering such questions as:
- why do we not use bcrypt here
- why do we not use HMAC to hash passwords
- why do we not store sensitive data on a cookie

I have been googling these questions while writing and they are not easy questions, but I can imagine people asking them.


--- 

>only the signature of the JWT that is hashed

this is actually specified in the JWT section, but agree it could be made clearer that the HMAC (message _authentication_ code) is used to verify the authenticity of a message, but is not intended to hide/encrypt the message at all. Will think about adding this.